fluidanimate/src-rl/tests/test_native_shell.cpp

```diff
@@ -0,0 +1,19 @@
+#include "raftlib_src.hpp"
```

Comment:
See changes committed for test_native_multi_shell.cpp, strdup might work now but it's b/c of headers included from somewhere else. Same for EXIT_SUCCESS/FAILURE.